db/filters/operations/apply.py

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env python3
```

Comment:
What is this empty file for?

Reply:
Oops.  That appears to be cruft from a branch change.  I must have had a file open in a different branch.